posthog/taxonomy/taxonomy.py

```diff
@@ -2136,6 +2148,7 @@ def decapitalize(match):
         "description": (
             f"{CORE_FILTER_DEFINITIONS_BY_GROUP['event_properties'][key]['description']} Captured at the start of the session and remains constant for the duration of the session."
         ),
+        "ignored_in_assistant": True,
```

Comment:
I changed this because the `.json` file included this but it would remove it on generation, so let's make the generation look like what we already had in the file